packages/core/src/Synchronizer.ts

```diff
@@ -638,13 +678,15 @@ export class Synchronizer extends EventEmitter {
     async handleStateTreeLeaf({ event, db, decodedData }: EventHandlerArgs) {
         const epoch = Number(decodedData.epoch)
         const index = Number(decodedData.index)
-        const attesterId = BigInt(decodedData.attesterId).toString()
-        const hash = BigInt(decodedData.leaf).toString()
-        if (
-            attesterId !== this.attesterId.toString() &&
-            this.attesterId !== BigInt(0)
-        )
-            return
+        const attesterId = toDecString(decodedData.attesterId)
+        const hash = toDecString(decodedData.leaf)
+        if (!this.attesterExist(attesterId)) return
```

Comment:
@vivianjeng if we have `_syncAll == true` will this event be processed?

Reply:
If _syncAll
The attester IDs will be in the list
So it will be processed